sphinx_gallery/tests/test_scrapers.py

```diff
@@ -15,6 +14,10 @@
 @pytest.fixture(scope='function')
 def gallery_conf(tmpdir):
     """Sets up a test sphinx-gallery configuration"""
+    # Skip if numpy not installed
+    pytest.importorskip("numpy")
+    import numpy as np
```

Comment:
You can just do:
```
    np = pytest.importorskip("numpy")
```
but really `np` isn't even used so this is fine
```suggestion
    pytest.importorskip("numpy")
```